assets/stylesheets/sift_classification.scss

```diff
@@ -15,77 +15,175 @@ $topic_list: 'General', 'Bullying', 'Fighting', 'PII', 'Sexting', 'Vulgar', 'Dru
   }
 }
 
+// Normal Sift risk colours
+:root {
+  --risk-0: #CCCCCC;
+  --risk-1: #AEAEAE;
+  --risk-2: #919191;
+  --risk-3: #747474;
+  --risk-4: #575757;
+  --risk-5: #3A3A3A;
+  --risk-6: #1D1D1D;
+  --risk-7: #000000;
+}
 
-.sift-classification-unit {
-  background: #e6f2ff;
+// Colourblind palette colours
+:root {
+  --risk-4-bg: rgb(204, 121, 167);
+  --risk-4-fg: black;
 
-  padding: 3px 6px;
-  border: solid 1px grey;
-  border-radius: 9px;
+  --risk-5-bg: rgb(240, 228, 66);
+  --risk-5-fg: black;
+
+  --risk-6-bg: rgb(230, 159, 0);
+  --risk-6-fg: black;
+
+  --risk-7-bg: rgb(213, 94, 0);
+  --risk-7-fg: black;
+
+}
 
+
+%sift-classification-base {
   font-size: 1em;
-  font-weight: bold;
-  color: white;
+  //color: white;
   text-shadow: 0 -1px 0 rgba(0, 0, 0, 0.25);
   white-space: nowrap;
   vertical-align: baseline;
-  background-color: #999999;
+  //background-color: #999999;
   display: inline-block;
   margin-bottom: 7px;
 }
 
-.sift-classification-unit + .sift-classification-unit {
-  margin-left: 2px;
+//////////////////////////////////////////////////
+//
+// Topic Badges
+//
+//////////////////////////////////////////////////
+.sift-topic {
+
+  @extend %sift-classification-base;
+
+  padding: 3px 6px;
+
+  color: white;
+  background-color: #999999;
```

Comment:
I suspect this is not theme friendly in Discourse. Some forums are dark, some are light. It would be good to use discourse's variables and helpers to set your colors.

Reply:
Fixed